handbook/engineering/code-intelligence/code-walkthrough.md

```diff
@@ -0,0 +1,51 @@
+# Code walkthrough
+
+The following page details how we approach the technical hiring within the Code Intelligence team instead of take home projects.
```

Comment:
```suggestion
The following page details how we approach technical hiring within the Code Intelligence team instead of take home projects.
```

Reply:
Still something feels confusing with this sentence. Proposed formulation

>  The following page details how the Code Intelligence team does a code walkthrough interview as a replacement for take home projects.

Reply:
+1 to @olafurpg's suggestion